### DISCUSSION TITLE
Triple-click to select paragraph?

### DISCUSSION THREAD

**Discussion by gzko** [sumatrapdfreader/sumatrapdf#3280] [Ideas]
It would be very convenient for me. Especially with QTranslate. Maybe someone else?

**Comment by GitHubRulesOK** [comment]
no way to decide what's a paragraph when clicking in PDF
 here is the end of one paragraph ( the full stop =`(.) Tj`) and start of another (the 1 in `(1) Tj`)
```
BT
/F1 9.6 Tf
11.0399999999999991 TL
0. 0. 0. rg
339.640625 620.1962499999999636 Td
(.) Tj
ET
0. w
0. 0. 0. rg
1. G
0. w
0 j
0. 0. 0. rg
0. w
BT
/F1 9.6 Tf
11.0399999999999991 TL
0. 0. 0. rg
50. 595.2400000000000091 Td
(1) Tj
ET
0. w
0. 0. 0. rg
1. G
0. w
0 j
0. 0. 0. rg
0. w
```
There is no end of paragraph there is no end of line they flow into each other just like any other two letters, what is the difference is the value of the down shift so one line is above the other (620.1962499999999636 is above 595.2400000000000091 PDFs are stacked from bottom of page.), but that applies to every line. as a Human with my Hind sight I can say that (1) at `50.` and 24.956249999 lower must heuristically by local page dynamics and human conventions be potentially start of a paragraph, but every line on that page starts at the same 100 twips (17.638 mm) value with different spacings.
![image](https://user-images.githubusercontent.com/12754913/219660635-f9a351c2-2a24-4d47-bc0d-c8308684d1c0.png)
